Examples/Python/python/acts/examples/simulation.py

```diff
@@ -378,8 +427,6 @@ def addFatras(
         the output folder for the Csv output, None triggers no output
     outputDirRoot : Path|str, path, None
         the output folder for the Root output, None triggers no output
-    rnd : RandomNumbers, None
-        random number generator
```

Comment:
should we keep this even tho it is not very informative?

Reply:
leftover as with the last PR